CIP-0030/README.md

```diff
@@ -1,72 +1,92 @@
 ---
 CIP: 30
 Title: Cardano dApp-Wallet Web Bridge
-Authors: rooooooooob
-Comments-URI: https://github.com/cardano-foundation/CIPs/pull/88
-Status: Draft
-Type: Standards
+Status: Active
+Category: Wallets
+Authors:
+  - rooooooooob
+Implementors:
+  - Begin <https://begin.is/>
+  - Eternl <https://eternl.io/>
+  - Flint <https://flint-wallet.com/>
+  - GeroWallet <https://www.gerowallet.io/>
+  - Lace <https://www.lace.io/>
+  - Nami <https://namiwallet.io/>
+  - NuFi <https://nu.fi/>
+  - RayWallet <https://raywallet.io/>
+  - Yoroi <https://yoroi-wallet.com/>
+Discussions:
+  - https://github.com/cardano-foundation/CIPs/pull/88
+  - https://github.com/cardano-foundation/CIPs/pull/148
+  - https://github.com/cardano-foundation/CIPs/pull/151
+  - https://github.com/cardano-foundation/CIPs/pull/183
+  - https://github.com/cardano-foundation/CIPs/pull/208
+  - https://github.com/cardano-foundation/CIPs/pull/323
+  - https://github.com/cardano-foundation/CIPs/issues/169
+  - https://github.com/cardano-foundation/CIPs/issues/178
+  - https://github.com/cardano-foundation/CIPs/issues/204
+  - https://github.com/cardano-foundation/CIPs/issues/253
+  - https://github.com/cardano-foundation/CIPs/issues/386
+  - https://github.com/cardano-foundation/CIPs/issues/404
+  - https://github.com/cardano-foundation/CIPs/issues/419
 Created: 2021-04-29
 License: CC-BY-4.0
 ---
 
-# Abstract
+## Abstract
 
 This documents describes a webpage-based communication bridge allowing webpages (i.e. dApps) to interface with Cardano wallets. This is done via injected javascript code into webpages. This specification defines the manner that such code is to be accessed by the webpage/dApp, as well as defining the API for dApps to communicate with the user's wallet. This document currently concerns the Shelley-Mary era but will have a second version once Plutus is supported. This specification is intended to cover similar use cases as web3 for Ethereum or [EIP-0012](https://github.com/ergoplatform/eips/pull/23) for Ergo. The design of this spec was based on the latter.
 
-
-# Motivation
+## Motivation: why is this CIP necessary?
 
 In order to facilitate future dApp development, we will need a way for dApps to communicate with the user's wallet. While Cardano does not yet support smart contracts, there are still various use cases for this, such as NFT management. This will also lay the groundwork for an updated version of the spec once the Alonzo hardfork is released which can extend it to allow for Plutus support.
 
+## Specification
 
+### Data Types
 
-# Specification
-
-## Version
-
-The API specified in this document will count as version 0.1.0 for version-checking purposes below.
-
-## Data Types
-
-### Address
+#### Address
 
 A string representing an address in either bech32 format, or hex-encoded bytes. All return types containing `Address` must return the bech32 format, but must accept either format for inputs.
```

Comment:
Might be out of scope of this PR, but if we do this:
> Which is why I am proposing to freeze the current API

Then we should do this:
```suggestion
A string representing an address in either bech32 format, or hex-encoded bytes. All return types containing `Address` must return the hex-encoded bytes format, but must accept either format for inputs.
```
Because it's how the cip is implemented in practice

Reply:
Sounds reasonable to me.